java/src/jmri/jmrix/openlcb/OlcbPowerManager.java

```diff
@@ -0,0 +1,98 @@
+package jmri.jmrix.openlcb;
+
+import jmri.PowerManager;
+import jmri.managers.AbstractPowerManager;
+import jmri.jmrix.can.CanSystemConnectionMemo;
+
+import org.openlcb.EventID;
+import org.openlcb.OlcbInterface;
+import org.openlcb.implementations.BitProducerConsumer;
+import org.openlcb.implementations.EventTable;
+import org.openlcb.implementations.VersionedValueListener;
+
+/**
+ * Extend jmri.managers.AbstractPowerManager for OpenLCB controls.
+ *
+ * @author Bob Jacobsen Copyright (C) 2025
+ */
+public final class OlcbPowerManager extends AbstractPowerManager<CanSystemConnectionMemo> {
+
+    OlcbAddress addrOn  = new OlcbAddress(new EventID("01.00.00.00.00.00.FF.FE"));
+    OlcbAddress addrOff = new OlcbAddress(new EventID("01.00.00.00.00.00.FF.FF"));
```

Comment:
Can these be changed? If not, I recommend to make them final.

Reply:
Why having them as non final if no one is expected to change them?

final OlcbAddress addrOn = ...